validate.lic

```diff
@@ -117,6 +117,13 @@ class DRYamlValidator
             .each { |skill_name| error("Invalid crafting_training_spells: skill name '#{skill_name}'. Valid skill names are '#{valid_crafting_training_spell_skills}'") }
   end
 
+  def assert_that_crafting_training_spells_have_no_cambrinth(settings)
+    return unless settings.crafting_training_spells
+
+    settings.crafting_training_spells.select { |_spell, data| data['cambrinth'] }
+            .each { |_spell, _data| warn("#{_name} in crafting_training_spells uses cambrinth which increases crafting time, you may want to remove it.") }
```

Comment:
❌  By convention, an underscore in a variable name means it is not used. Seeing both `_spell` and `_data` made me wonder what was being reported in the `.each`. I don't think `_name` is defined.

Reply:
I mean it worked, I tested it, but I don't know how to fix it properly so I'll just revert that one.

Reply:
I think the test was reasonable. I suspect `_name` should have been `_data['name']`.